bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/VoiceManager.java

```diff
@@ -469,6 +470,26 @@ public TTSService getTTS() {
     }
 
     /**
+     * Retrieves a TTS service with the given id.
+     *
+     * @param id the id of the TTS service
+     *
+     * @return a TTS service or null, if no service with this id exists
+     */
+    public TTSService getTTS(String id) {
+        return ttsServices.get(id);
+    }
+
+    /**
+     * Retrieves all TTS services.
+     *
+     * @return a set of TTS services
```

Comment:
* it is not necessary a "set" but a collection
* should we add the information, that the returned collection will throw an exception if someone tried to modify it (so add it is an unmodifiable collection)